### ISSUE TITLE
Wasm decoding failedResult = expected version 01 00 00 00, found 0d 00 00 00 @+4

### ISSUE THREAD

**Issue by izengliang** [ewasm/wast2wasm#7]
## example.wast
```
(module 
    (func (export "test") (param $a i32) (result i32) 
        (i32.add
             (get_local $a)
            (i32.const 3)  
        )
    )
) 
```

## build.js
```
...
let buf = fs.readFileSync("example.wast");
let result = await wast2wasm(buf.toString(),true);
console.log(result.log);
fs.writeFileSync("example.wasm",Buffer.from(result.buffer));
...
```

###  run build.js result
```
0000000: 0061 736d                                 ; WASM_BINARY_MAGIC
0000004: 0d00 0000                                 ; WASM_BINARY_VERSION
; section "TYPE" (1)
0000008: 01                                        ; section code
0000009: 00                                        ; section size (guess)
000000a: 01                                        ; num types
; type 0
000000b: 60                                        ; func
000000c: 01                                        ; num params
000000d: 7f                                        ; i32
000000e: 01                                        ; num results
.....
```

### example.html 
```
CompileError: WebAssembly.instantiate(): Wasm decoding failedResult = expected version 01 00 00 00, found 0d 00 00 00 @+4
```

But, use https://github.com/WebAssembly/wabt tool  is ok.

**Comment by wanderer** [comment]
@liangzeng how are you compiling `example.wasm`? it looks like it using the old 0xd version. You probably need to compile with a new version.

**Comment by izengliang** [comment]
@wanderer i know, wast2wasm/wabt version is old , need  update.

**Comment by wanderer** [comment]
@liangzeng thanks! this is now fix in 0.0.1
